packages/core/scripts/local/CalculateContractBytecode.js

```diff
@@ -1,23 +1,35 @@
 // This simple script tells you how big your contract byte code is and how much you have until you exceed
 // the current block limit as defined by EIP170. This script should be run from the /core directory.
 // To run the script navigate to /core and then run:
-// truffle exec -c ./scripts/local/CalculateContractBytecode.js --contract Voting --network test
+// yarn truffle compile && yarn buidler compile && yarn truffle exec --network test ./scripts/local/CalculateContractBytecode.js --contract Voting
 // where voting is the name of the contract you want to check.
 
 const argv = require("minimist")(process.argv.slice(), { string: ["contract"] });
 
+function printBytecodeInfo(path) {
+  // Load contracts into script and output info.
+  const obj = require(path);
+  const byteCodeSize = (obj.deployedBytecode.length - 2) / 2;
+  const remainingSize = 2 ** 14 + 2 ** 13 - byteCodeSize;
+  console.log("Contract is", byteCodeSize, "bytes in size.");
+  console.log("This leaves a total of", remainingSize, "bytes within the EIP170 limit.");
+}
+
 module.exports = async function(callback) {
   if (!argv.contract) {
     console.log("Please enter the contract name as a parameter as `--contract <name>`.");
     callback();
   }
 
-  // Load contracts into script and output info.
-  console.log("loading", argv.contract + ".json");
-  let obj = require("./../../build/contracts/" + argv.contract + ".json");
-  const byteCodeSize = (obj.deployedBytecode.length - 2) / 2;
-  const remainingSize = 2 ** 14 + 2 ** 13 - byteCodeSize;
-  console.log("Contract is", byteCodeSize, "bytes in size.");
-  console.log("This leaves a total of", remainingSize, "bytes within the EIP170 limit.");
+  // Truffle
+  console.group("Truffle Compilation Output");
+  printBytecodeInfo("./../../build/contracts/" + argv.contract + ".json");
+  console.groupEnd();
+
+  // Buidler
```

Comment:
what is the true bytecode output? one is right and the other is wrong...how do we know the correct outcome? Both Truffle and builder are using the same optimizer settings and compiler version.

Reply:
Agreed that one should, in theory, be "wrong," but I looked through the settings and couldn't find anything to differentiate them. I wonder if there's some difference in the way the files are passed to solc that causes the difference...?

Reply:
As mentioned offline, removing the unnecessary import fixed this.

Reply:
I still think having both in here is useful, though, in case they differ again in the future. Is that cool with you?